cmake/projects/spdlog/hunter.cmake

```diff
@@ -119,6 +119,17 @@ hunter_add_version(
     5dd633311d1aec79404d6694b452745b80f1a7d1
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    spdlog
+    VERSION
+    "1.8.0-p0"
+    URL
+    "https://github.com/cpp-pm/spdlog/archive/v1.8.0-p0.tar.gz"
```

Comment:
Made a comment in the spdlog fork, but the line `find_package(fmt 5.3.0 REQUIRED CONFIG)` will fail with updated versions of fmt in Hunter. Can you remove the version configuration, just like the older `spdlog` versions?

Reply:
Yes I've created a corresponding PR: https://github.com/cpp-pm/spdlog/pull/2 . I hope that's the way to do it? Or should I create a new branch on cpp-pm/spdlog?

Reply:
yup that's good

Reply:
@craffael https://github.com/cpp-pm/spdlog/releases/tag/v1.8.0-p1

Reply:
okay I've updated everything so that v1.8.0-p1 is used. Travis and Appveyor work as expected (I've updated the links in the description of the PR). However, the new Github Action workflow seems to fail in one instance. Can I somehow disable that check?

Reply:
all good for now